python/lsst/ctrl/stats/data/dbEntry.py

```diff
@@ -33,15 +32,33 @@ def __init__(self, dbList):
         """
         # the DAG node name
```

Comment:
I'd rather the constructor arguments were actually documented by stating what each element of `dbList` refers to in the docstring, rather than inline comments saying that element 0 is the DAG node name. Also, this should go in the class docstring, not the `__init__` docstring: https://developer.lsst.io/docs/py_docs.html#classes-methods-and-functions